Snippets/Snippets_5/Outbox/SqlServer/ConnectionStrings.xml

```diff
@@ -1,6 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
 <!--startcode OutboxSqlServerConnectionStrings-->
 <configuration>
+  <configSections>
+    <section name="AuditConfig" type="NServiceBus.Config.AuditConfig, NServiceBus.Core"/>
```

Comment:
my point is this snippet cannot possibly be used in any sample. if u want audit configured it needs to be in this file https://github.com/Particular/docs.particular.net/blob/master/samples/outbox/sqltransport-nhpersistence/Version_2.1/Receiver/app.config